espanol/explosion/explosion.md

```diff
@@ -0,0 +1 @@
+El cuarto había quedado en escombros y el consejero había desaparecido, el propio Beatito fue el único que no sufrió ningún cambio.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.